--- lib.rs	2023-01-09 21:03:20.692716983 +0000
+++ lib.rs_	2023-01-09 21:04:00.553067917 +0000
@@ -1,3 +1,4 @@
+use rayon::prelude::*;
 use rayon::slice::ParallelSliceMut;
 
 use crate::*;
@@ -23,7 +24,7 @@
                     (idx, (a + b).powf(0.5))
                 })
                 .collect();
-            distances.sort_unstable_by(|(_, a), (_, b)| {
+            distances.par_sort_unstable_by(|(_, a), (_, b)| {
                 a.partial_cmp(b).expect("partial_cmp found NaN")
             });
             let ballot: Vec<_> =
