--- lib.rs	2023-01-09 20:54:23.817814250 +0000
+++ lib.rs_	2023-01-09 20:58:12.909787252 +0000
@@ -23,7 +23,7 @@
                     (idx, (a + b).powf(0.5))
                 })
                 .collect();
-            distances.par_sort_unstable_by(|(_, a), (_, b)| {
+            distances.sort_unstable_by(|(_, a), (_, b)| {
                 a.partial_cmp(b).expect("partial_cmp found NaN")
             });
             let ballot: Vec<_> =
